lib/rubocop/cop/style/leading_comment_space.rb

```diff
@@ -27,6 +29,12 @@ def autocorrect(comment)
           hash_mark = range_between(b, b + 1)
           ->(corrector) { corrector.insert_after(hash_mark, ' ') }
         end
+
+        private
+
+        def config_ru?(file_path)
+          File.basename(file_path).eql?('config.ru')
```

Comment:
Now I'm a bit worried that this filename check might slow down the cop. Can you do some benchmarks before and after running only this cop on a big project?